.js-beautify.json

```diff
@@ -1,19 +0,0 @@
-{
-  "indent_size": "2",
-  "indent_char": " ",
-  "max_preserve_newlines": "5",
-  "preserve_newlines": true,
-  "keep_array_indentation": false,
-  "break_chained_methods": false,
-  "indent_scripts": "normal",
-  "brace_style": "collapse",
-  "space_before_conditional": true,
-  "unescape_strings": false,
-  "jslint_happy": false,
-  "end_with_newline": false,
```

Comment:
Can we change this to true? I think this is what most editors want.

Reply:
I deliberately made no changes to the existing code style definition, since style discussions can be a rabbit hole. I've opened https://github.com/dekkerglen/CubeCobra/issues/450 to track changes to this definition outside of this pull request, which simply moves the existing definition.

Reply:
Fair enough